lib/zclient.c

```diff
@@ -3664,7 +3664,7 @@ int lm_get_label_chunk(struct zclient *zclient, uint8_t keep, uint32_t base,
 	if (zclient_debug)
```

Comment:
maybe this debug should be moved to after the arg-check, so no one is tempted to de-ref 'zclient' too early?